syntax = "proto3";

package myrtlespeech.protos;

import "myrtlespeech/protos/rnn.proto";

// Configuration for Transducer Prediction Network
// https://arxiv.org/pdf/1211.3711.pdf
message TransducerPredictNet {
  PredNN pred_nn = 1;
}

message PredNN {
  // Currently PredNN must be an RNN only but
  // this may change in the future
  RNN rnn = 1;
}
